ironfish/src/rpc/routes/wallet/exportAccount.ts

```diff
@@ -29,6 +31,7 @@ export const ExportAccountResponseSchema: yup.ObjectSchema<ExportAccountResponse
       .object({
         name: yup.string().defined(),
         spendingKey: yup.string().defined(),
```

Comment:
Should `spendingKey` not be not required for defined? If it can be null on return